Submitted By:            Douglas R. Reno <renodr at linuxfromscratch dot org>
Date:                    2024-02-26
Initial Package Version: 23.08.5
Upstream Status:         Rejected (Qt6 port is primary now)
Origin:                  Self
Description:             Fixes building Gwenview with recent kImageAnnotator
                         and kColorPicker versions by adjusting their paths
                         and FOUND variables so that the correct libraries are
                         selected.

--- gwenview-23.08.5.orig/CMakeLists.txt	2024-02-26 15:02:29.702754535 -0600
+++ gwenview-23.08.5/CMakeLists.txt	2024-02-26 15:09:48.012866877 -0600
@@ -166,11 +166,11 @@ if(NOT WITHOUT_X11)
 endif()
 
 if (QT_MAJOR_VERSION STREQUAL "5")
-    find_package(kImageAnnotator)
-    set_package_properties(kImageAnnotator PROPERTIES URL "https://github.com/ksnip/kImageAnnotator" DESCRIPTION "The kImageAnnotator library provides tools to annotate" TYPE REQUIRED)
-    if(kImageAnnotator_FOUND)
-        set(KIMAGEANNOTATOR_FOUND 1)
-        find_package(kColorPicker REQUIRED)
+    find_package(kImageAnnotator-Qt5)
+    set_package_properties(kImageAnnotator-Qt5 PROPERTIES URL "https://github.com/ksnip/kImageAnnotator" DESCRIPTION "The kImageAnnotator library provides tools to annotate" TYPE REQUIRED)
+    if(kImageAnnotator-Qt5_FOUND)
+        set(kImageAnnotator_FOUND 1)
+        find_package(kColorPicker-Qt5 REQUIRED)
         if(NOT kImageAnnotator_VERSION VERSION_LESS 0.5.0)
             set(KIMAGEANNOTATOR_CAN_LOAD_TRANSLATIONS 1)
         endif()
